src/NUnitEngine/nunit.engine.tests/Internal/PathUtilTests.cs

```diff
@@ -86,7 +86,7 @@ public void Canonicalize()
 				PathUtils.Canonicalize( @"folder1\folder2\..\..\..\file.tmp" ) );
 		}
 
-#if !NETCOREAPP1_1
+#if !NETCOREAPP1_1 && !NETCOREAPP2_0
```

Comment:
I suggest we `#if` out the actual method inline with the test being excluded here. Nothing actually uses this method in the .NET Standard builds.

Reply:
Ahh, it's the PlatormAttribute that's the issue. 

I'll try upgrading .NET Core 2.0 to 2.1 again. Last time I did that it caused a problem somewhere along the line, but I can't quite remember what.